### PR TITLE
[SUMMIT] - Add new speakers

### PR DESCRIPTION
**Pull request by illia-stv**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -591,6 +591,66 @@ <h4>React Remixed</h4>
...
+                      <a
+                        href="https://twitter.com/przemyslawjanp"
+                        target="_blank"
+                        title="Twitter profile - Przemysław Beigert"
+                        >#przemyslawjanp</a
+                      >
+                    </div>
+                  </div>
+
+                  <h4>Advanced TypeScript</h4>
```
Comment (JKrymarys) [inline src/index.html]: Is it worth learning Vue in 2022 and why not?

File: src/index.html [hunk 2]
```diff
@@ -591,6 +591,66 @@ <h4>React Remixed</h4>
...
+                  <p class="talks-box__desc">
+                    TypeScript become extremely popular tool but how well you know him? Probably you
```
Comment (JKrymarys) [inline src/index.html]: This is not another introduction to the Vue. I'm gonna focus on differences between Vue, React and Angular in topics like: e.g.: communication, forms, storage, testing. All with code examples. Base on that I'll tell when Vue will be good choose and why invest time in learning it isn't good idea.

File: src/index.html [hunk 3]
```diff
@@ -591,6 +591,66 @@ <h4>React Remixed</h4>
...
+                        href="https://twitter.com/KonradHanus"
+                        target="_blank"
+                        title="Twitter profile - Konrad Hanus"
+                        >#KonradHanus</a
+                      >
+                    </div>
+                  </div>
+
+                  <h4>Our approach to unifying company products with shareable library</h4>
+                  <p class="talks-box__desc">
```
Comment (JKrymarys) [inline src/index.html:648]: Have you ever played Mario Bros?

Let's drop our adult belief that playing is something frivolous and a waste of time.

Let's have fun, write the game in JS/React
topics:
🍄 the technique of creating gravity
👨🏻 player creation
🕹️ platforms
🐢 parallax effect
⭐ level creation

File: src/index.html [hunk 4]
```diff
@@ -591,6 +591,66 @@ <h4>React Remixed</h4>
...
+                      <a
+                        href="https://twitter.com/KonradHanus"
+                        target="_blank"
+                        title="Twitter profile - Konrad Hanus"
+                        >#KonradHanus</a
+                      >
+                    </div>
+                  </div>
+
+                  <h4>Our approach to unifying company products with shareable library</h4>
```
Comment (JKrymarys) [inline src/index.html]: How to make an engine for a side scroller video game using HTML5 Cavnas and React

**Review by JKrymarys — CHANGES_REQUESTED** [review]
I think that you looked at the wrong row :P In comments I added the proper titles and messages

**Comment by naugtur** [comment]
To nadal wisi z pomieszanymi opisami? :open_mouth:

**Pull request merged**